offseason-bot/src/main/java/frc/robot/autos/auto_state_machines/StationAndLollipop5pcAutoTrailblazer.java

```diff
@@ -0,0 +1,301 @@
+package frc.robot.autos.auto_state_machines;
```

Comment:
please decide the best way to write this auto and commit to one implementation. you can do a mix of trailblazer and pid to point. whatever makes sense